basics/answers.md

```diff
@@ -295,6 +306,26 @@ isn't exactly like mine but it works just fine it's ok, and you can
         converted_numbers.append(int(number))
     print(converted_numbers)
     ```
+5. We actually use the first `for` loop to loop through the number of rows needed.Now,
```

Comment:
This line seems to refer to the first line of the code below it. If you want to write the explanation like that, then show it first, so that it can be read easily from top to bottom.